src/Core/Domain/MerchandiseReturn/ValueObject/MerchandiseReturnStateId.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\MerchandiseReturn\ValueObject;
+
+use PrestaShop\PrestaShop\Core\Domain\MerchandiseReturn\Exception\MerchandiseReturnOrderStateConstraintException;
+
+/**
+ * @todo I don't know whether this should be MerchandiseReturnStateId or OrderReturnStateId
```

Comment:
I think both are find because equivalent 😄 

So in order to avoid bothering you let's keep MerchandiseReturnStateId